client/src/pages/live-Invocations/LiveInvocationUpperBar.tsx

```diff
@@ -92,6 +93,8 @@ export const InvocationUpperBar = ({
       <div className="flex flex-row items-center space-x-2">
         {component}
         <div className="flex flex-row items-center w-28">
+        <IoMdArrowRoundBack className="text-2xl mr-1" style={{cursor:'pointer'}} onClick={()=>{ navigator(-1)}}/> 
```

Comment:
please don't mix inline style and tailwindcss. use cursor-pointer class.

Reply:
Could you change it to this component - <FaArrowLeftLong />?

Reply:
<FaArrowLeftLong />
if you mean for icon it's does not exist.